app/src/main/java/agency/five/tmdb/MovieRepository.kt

```diff
@@ -0,0 +1,22 @@
+package agency.five.tmdb
+
+import agency.five.tmdb.ui.components.MovieItemViewState
+import kotlinx.coroutines.flow.Flow
+
+interface MovieRepository {
+    fun getPopularMovies(): Flow<List<List<MovieItemViewState>>>
```

Comment:
Use `Repository` level models (List<Movie>, List<Actor> ...) instead of view-state models.
View State models should be used only in VM and views.